common/src/main/java/io/netty/util/concurrent/DefaultPromise.java

```diff
@@ -31,8 +31,8 @@
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 
 public class DefaultPromise<V> implements Promise<V> {
-    private static final InternalLogger logger = InternalLoggerFactory.getInstance(DefaultPromise.class);
-    private static final InternalLogger rejectedExecutionLogger =
+    private static final InternalLogger LOGGER = InternalLoggerFactory.getInstance(DefaultPromise.class);
+    private static final InternalLogger REJECTED_EXECUTION_LOGGER =
```

Comment:
Loggers are not constants so these fields should remain camel-cased.

Reply:
I think the convention is to use this shouting case for static final fields regardless of the mutability of the referred data.

Reply:
The convention at Netty as far as I know is to use uppercase snake case for constants only. Modern IDEs do syntax highlighting for static members, so just being static doesn't mean it deserves upper casing in my opinion.